### PR TITLE
Text component

### PR DESCRIPTION
**Pull request by Zarthia**

### CODE REVIEW

**Comment by Tuxosaurus** [comment]
Specs, for reference:
typography_design_system_1_8_3

File: src/text/index.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  readonly children: string | number | React.ReactNode,
+  readonly display?: TextDisplayType,
+  readonly div?: boolean,
+}
+
+const Text = ({
+  className,
+  children,
+  display = TextDisplayType.BODY,
+  div = false,
```
Comment (loick) [inline src/text/index.tsx]: can it only be a div? No bold, italic etc.?

File: src/text/style.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import css from 'styled-jsx/css'
+import { color, font, fontWeight } from '_utils/branding'
+
+export default css`
+  .kirk-text {
```
Comment (loick) [inline src/text/style.ts]: remove this?

File: src/text/index.tsx (hunk 1 above)
Comment (Tuxosaurus) [reply src/text/index.tsx]: This is handled by "display", which corresponds to the different types of text shown in the design system

File: src/text/index.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+export enum TextDisplayType {
+  BODY = 'body',
+  BODYSTRONG = 'bodyStrong',
+  BUTTON = 'button',
+  CAPTION = 'caption',
+  DISPLAY1 = 'display1',
+  DISPLAY2 = 'display2',
+  SUBHEADER = 'subheader',
+  SUBHEADERSTRONG = 'subheaderStrong',
+  TITLE = 'title',
```
Comment (loick) [inline src/text/index.tsx]: Move this in the Title component x2

File: src/text/style.ts [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    line-height: ${font.base.lineHeight};
+  }
+
+  .kirk-text-caption {
+    color: ${color.secondaryText};
+    font-size: ${font.s.size};
+    line-height: ${font.s.lineHeight};
+  }
+
+  .kirk-text-display1 {
```
Comment (nickoola) [inline src/text/style.ts]: If this display class is related to the size it should be named like .kirk-text-xxl and .kirk-text-xl

**Pull request merged**